### ISSUE TITLE
Can't see AUDIO Inputs

### ISSUE THREAD

**Issue by mrdrogdrog** [mvdwetering/yamaha_ynca#230]
**Describe the bug**
I installed the integration and it found my RX-V475 but it doesn't detect the Audio-Input. I checked the integration configuration and it doesn't list them as well. The Yamaha App shows them.

**To Reproduce**
1. Have a Yamaha RX-V475
2. Add Integration
3. Check the Input list

**Expected behavior**
The Audio Input should be visible.

**Diagnostics**

[config_entry-yamaha_ynca-d0e73aa37e8a9680d55ff9880d63ff9e.json.txt](https://github.com/mvdwetering/yamaha_ynca/files/13951244/config_entry-yamaha_ynca-d0e73aa37e8a9680d55ff9880d63ff9e.json.txt)

**Logging**
```
Logger: ynca.enums
Source: /usr/local/lib/python3.11/site-packages/ynca/enums.py:187
First occurred: 14:55:50 (4 occurrences)
Last logged: 15:14:01

Unknown value 'AUDIO' in Input
```

**Environment**
Provide details about the versions you are using, which helps to reproduce and find the issue quicker.

- Home Assistant Core version: 2024.1.2
- Home Assistant operating environment (OS/Container/Supervised/Core): Docker-Image
- Integration version: [v7.5.1](https://github.com/mvdwetering/yamaha_ynca/releases/tag/v7.5.1)

**Comment by mvdwetering** [comment]
This is the first time I see a receiver with an input called "AUDIO", usually there seem to be more and they are called "AUDIO1",  "AUDIO2" etc...

Adding the AUDIO input itself is not that hard, but the complicated part seems to be detecting if the input is supported by the receiver. I detect availablity by asking for the names of all inputs on the receiver and it will give a list of names for all supported inputs. However in the diagnostics you provided I don't see a name for the "AUDIO" input.

I would prefer to autodetect the availablility of the input instead of having to code something special for it per receiver.
Could you try the following experiments to see if there is something I can use?

* Change the name of the AUDIO input through the menus on the receiver (not using an app). Just changing a letter is fine and then exit the menus.
* In the Home Assistant developer tools on the "services" tab use the "Yamaha (YNCA): Send raw YNCA command" service to send the commands below. These can be pasted all at once into the "Raw YNCA data" field. Make sure to select your device as target before clicking "Call service"

```
@SYS:INPNAMEAUDIO=?
@SYS:INPNAMEAUDIO0=?
@SYS:INPNAMEAUDIO1=?
@SYS:TRIG1INPAUDIO=?
@SYS:TRIG1INPAUDIO0=?
@SYS:TRIG1INPAUDIO1=?
```

Afterwards please download the diagnostics file again and report back so I can see if there is something useful in there.
You can also change back the name of the audio input.

**Comment by mrdrogdrog** [comment]
Well. I have been surprised as well. I'm gonna test your proposal as soon as I have time.

**Comment by mrdrogdrog** [comment]
Here you go. Sry it took so long.
I didn't have to change the input names because all my inputs already have custom names.

[config_entry-yamaha_ynca-d0e73aa37e8a9680d55ff9880d63ff9e.json (2).txt](https://github.com/mvdwetering/yamaha_ynca/files/14071788/config_entry-yamaha_ynca-d0e73aa37e8a9680d55ff9880d63ff9e.json.2.txt)

**Comment by mvdwetering** [comment]
When a setting/value changes on the receiver it usually sends out a message with the change. The message for reporting the change is typically the same as for getting the value for such a setting, which is how I detect if features are supported. So changing of the inputname was intended to figure out if there was a message related to that specific input that I don't know of.

Based on the diagnostics it seems that the receiver responds differently for AUDIO1 (it says RESTRICTED instead of UNDEFINED) but it is in a way that I can't really handle. 

I guess the only way around this is to code a workaround specifically for the RX-V475 (and RX-V575/HTR-4066/HTR-5066 as they share the same firmware)

**Comment by mrdrogdrog** [comment]
Are you already working on this? Can I help you somehow?

**Comment by mvdwetering** [comment]
I started on it, but ran into some issues so couldn't finish it yet. Just need to find some time to work on it again.

**Comment by mvdwetering** [comment]
FYI this is fixed in v7.6.0

**Comment by mvdwetering** [comment]
Closing since it has bee fixed
